autoload/pythonimports.vim

```diff
@@ -7,17 +7,24 @@ function! pythonimports#filename2module(filename)
   let pkg = fnamemodify(a:filename, ":p")
   let root = fnamemodify(pkg, ":h")
 
+  "normalize  paths
+  let _tmp = []
+  for path in g:pythonPaths
+    call add(_tmp, expand(path:p))
+  endfor
+  let g:pythonPaths = _tmp
```

Comment:
Please don't overwrite the global.  As a user when I stick ~/src/myproj in it, I expect it to stay that way.  Better rename _tmp to l:pythonPaths or, on second thought, l:normalizedPaths, to avoid confusion.